blueprint/api.apib

```diff
@@ -1358,6 +1358,153 @@ The Stripe plan resource allows users to select a monthly donation amount for a
 
     + Attributes (Record Not Found Response)
 
+# Group Stripe Subscriptions
+
+This endpoint is for creating, editing and returning Stripe Subscriptions.
+
+Stripe Subscriptions allow us to charge a customer's card on a recurring basis. A subscription ties a customer to a particular plan.
```

Comment:
Prefer lowercase "subscriptions" in the descriptions here.

I would also state that our default here is monthly, that we're using $0.01 cent plans, and doing some X multiplier on top of that to set the actual amount.

I would also link to the Stripe docs from here.